include/openssl/ssl.h

```diff
@@ -2675,13 +2675,21 @@ OPENSSL_EXPORT int SSL_set1_groups_list(SSL *ssl, const char *groups);
 // https://datatracker.ietf.org/doc/html/draft-tls-westerbaan-xyber768d00
 #define SSL_GROUP_X25519_KYBER768_DRAFT00 0x6399
 
+// https://datatracker.ietf.org/doc/html/draft-kwiatkowski-tls-ecdhe-mlkem.html
+#define SSL_GROUP_SECP256R1_MLKEM768 0x11EB
+#define SSL_GROUP_X25519_MLKEM768    0x11EC
+
 // PQ and hybrid group IDs are not yet standardized. Current IDs are driven by
 // community consensus and are defined at
 // https://github.com/open-quantum-safe/oqs-provider/blob/main/oqs-template/oqs-kem-info.md
 #define SSL_GROUP_KYBER512_R3 0x023A
 #define SSL_GROUP_KYBER768_R3 0x023C
 #define SSL_GROUP_KYBER1024_R3 0x023D
 
+// https://datatracker.ietf.org/doc/html/draft-connolly-tls-mlkem-key-agreement.html
```

Comment:
Why pull these values from this IETF instead of the values defined in oqs-kem-info.md like Kyber? Is this IETF more widely accepted as authoritative?

Reply:
They are the same values. The [OQS codepoint for ML-KEM768 is `0x0768`](https://github.com/open-quantum-safe/oqs-provider/blob/main/ALGORITHMS.md?plain=1#L44) and `draft-connolly-tls-mlkem-key-agreement` also proposes [the same `0x0768` codepoint for the same ML-KEM768 algorithm](https://datatracker.ietf.org/doc/html/draft-connolly-tls-mlkem-key-agreement.html).

Generally, the draft RFC's define are considered more authoritative since only a few of the codepoints intended for standardization for internet-wide compatibility are turned into draft RFC's. Whereas LibOQS codepoints are defined for internal/experimental use that other libraries can use (but no long-term guarantees are made).